schemas/resources/schemas/orderoption/v1.0.0/ingest.md

```diff
@@ -0,0 +1,64 @@
+## <a name="orderoption"></a> Order Option
+
+#### <a name="provider-info-orderoption"></a> /providers/&lt;provider-id&gt;/orderoptions/&lt;native-id&gt;
+
+### <a name="create-update-orderoption"></a> Create / Update an Order Option
+
+Order option concepts can be created or updated by sending an HTTP PUT with the metadata to the URL `%CMR-ENDPOINT%/orderoption/<native-id>?provider=<provider-id>`. The response will include the [concept id](#concept-id) and the [revision id](#revision-id). The contents of the metadata is passed in the body of the request
+
+```
+curl -v -XPOST \
+-H "Content-Type:application/vnd.nasa.cmr.umm+json" \
+-H "Authorization: Bearer XXXX" \
+"%CMR-ENDPOINT%/orderoption/sampleNativeId?provider=PROV1" \
+-d @sampleOrderoption.json
+```
+
+#### Successful Response in XML
+```
+<?xml version="1.0" encoding="UTF-8"?><result><concept-id>OO1200000000-PROV1</concept-id><revision-id>1</revision-id><warnings></warnings><existing-errors></existing-errors></result>%
+```
+Subsequet ingests to a orderoption record will result in updates to it's metadata as well as increment the revision-id of the record.
+
+#### Successful Response in JSON
+
+By passing the option `-H "Accept: application/json"` to `curl`, one may
+get a JSON response:
+
+  {"concept-id":"OO1200000000-PROV1","revision-id":1,"warnings":null,"existing-errors":null}
+
+### <a name="delete-orderoption"></a> Delete a Order Option
+
+Order option metadata can be deleted by sending an HTTP DELETE to the URL `%CMR-ENDPOINT%/orderoption/<native-id>?provider=<provider-id>`. The response will include the [concept id](#concept-id) and the [revision id](#revision-id) of the tombstone.
+
+  curl -i -XDELETE \
```

Comment:
drop -i

Reply:
I'll go ahead and drop those on all the docs

Reply:
Though I would mentioned these curl flags are used in the same way in the Ingest api.md and search api.md